HealthcareSystem/PatientWebAppUnitTests/UnitTests/ExaminationControllerTests.cs

```diff
@@ -62,26 +64,173 @@ public void Get_existent_examination_by_non_patient_jmbg()
 
 
         [Fact]
-        public void Advanced_search_examination()
+        public void Advanced_search_examination_operator_and_anamnesis()
         {
-            ExaminationController examinationController = SetupExaminationController();
+            ExaminationService examinationService = new ExaminationService(_stubRepository.CreateExaminationStubRepository());
 
             var examinationSearchDTOValidObject = _objectFactory.GetExaminationSearchDTO().CreateValidTestObject();
-            var result = examinationController.AdvanceSearchExaminations(examinationSearchDTOValidObject);
+            var result = examinationService.AdvancedSearch(examinationSearchDTOValidObject);
 
-            Assert.True(result is OkObjectResult);
+            Assert.NotEmpty(result);
+        }
+
+        [Fact]
+        public void Advanced_search_examination_operator_or_anamnesis()
+        {
+            ExaminationService examinationService = new ExaminationService(_stubRepository.CreateExaminationStubRepository());
+            
+            var result = examinationService.AdvancedSearch(new ExaminationSearchDTO(jmbg: "1309998775018", startDate: new DateTime(2020, 10, 10), endOperator: LogicalOperator.Or, endDate: new DateTime(2020, 12, 12), doctorOperator: LogicalOperator.And, doctorSurname: null, anamnesisOperator: LogicalOperator.Or, anamnesis: "lklklkl"));
+
+            Assert.NotEmpty(result);
+        }
```

Comment:
This and the rest of the similar tests are alright, although they could be further improved. Have you considered using Theory, since you have many similar tests here? And I would maybe add more entities in the repo, as far as I can tell there's only one during these tests. Although that definitely would make coming up with the data and correctly determining what will be returned a lot more complicated, I think I don't mind if you leave these as they are.